Sources/NIOCore/ChannelPipeline.swift

```diff
@@ -1492,6 +1492,8 @@ public enum ChannelPipelineError: Error {
     case alreadyRemoved
     /// `ChannelHandler` was not found.
     case notFound
+    /// `ChannelHandler` is not auditable.
+    case notAuditable
```

Comment:
We're not going to be able to add a new error case here: we'll either need a new error, or we'll need to find a way to use one we already have.